Index: konsole/src/TerminalDisplay.h
===================================================================
--- konsole.orig/src/TerminalDisplay.h	2013-02-18 12:11:02.117782190 +0100
+++ konsole/src/TerminalDisplay.h	2013-02-18 12:14:25.000000000 +0100
@@ -405,7 +405,7 @@
         if (_bidiEnabled) {
             setLineSpacing(0);
         } else {
-            setLineSpacing(2);
+            setLineSpacing(0);
         }
 #endif
     }
