Index: konsole-4.8.80/src/main.cpp
===================================================================
--- konsole-4.8.80.orig/src/main.cpp	2012-05-24 08:12:37.000000000 +0100
+++ konsole-4.8.80/src/main.cpp	2012-06-06 14:30:31.999789567 +0100
@@ -176,6 +176,7 @@
     options.add("list-profile-properties",
                 ki18n("List all the profile properties names and their type"
                       " (for use with -p)"));
+    options.add("T <value>", ki18n("Debian policy compatibility, not used"));
     options.add("p <property=value>",
                 ki18n("Change the value of a profile property."));
     options.add("!e <cmd>",
