--- a/src/main.cpp
+++ b/src/main.cpp
@@ -105,6 +105,7 @@
     // TODO - Document this option more clearly
     options.add("p <property=value>",ki18n("Change the value of a profile property."));
     options.add("list-profile-properties", ki18n("List all the profile properties names and their type (for use with -p)"));
+    options.add("T <value>", ki18n("Debian policy compatibility, not used"));
     options.add("!e <cmd>",ki18n("Command to execute"));
     options.add("+[args]",ki18n("Arguments passed to command"));
 }
