Index: konsole/src/main.cpp
===================================================================
--- konsole.orig/src/main.cpp	2013-04-03 19:44:13.305306792 +0000
+++ konsole/src/main.cpp	2013-04-03 19:44:13.301306811 +0000
@@ -184,6 +184,7 @@
                       " (for use with -p)"));
     options.add("p <property=value>",
                 ki18n("Change the value of a profile property."));
+    options.add("T <value>", ki18n("Debian policy compatibility, not used"));
     options.add("!e <cmd>",
                 ki18n("Command to execute. This option will catch all following"
                       " arguments, so use it as the last option."));
