Index: konsole/src/main.cpp
===================================================================
--- konsole.orig/src/main.cpp	2015-05-27 18:02:19.804651972 +0200
+++ konsole/src/main.cpp	2015-05-27 18:02:19.800652134 +0200
@@ -197,6 +197,7 @@
                       " (for use with -p)"));
     options.add("p <property=value>",
                 ki18nc("@info:shell", "Change the value of a profile property."));
+    options.add("T <value>", ki18n("Debian policy compatibility, not used"));
     options.add("!e <cmd>",
                 ki18nc("@info:shell", "Command to execute. This option will catch all following"
                       " arguments, so use it as the last option."));
