--- a/src/main.cpp
+++ b/src/main.cpp
@@ -184,6 +184,7 @@ void fillCommandLineOptions(KCmdLineOpti
                       " (for use with -p)"));
     options.add("p <property=value>",
                 ki18n("Change the value of a profile property."));
+    options.add("T <value>", ki18n("Debian policy compatibility, not used"));
     options.add("!e <cmd>",
                 ki18n("Command to execute. This option will catch all following"
                       " arguments, so use it as the last option."));
